bower.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "skatejs-types",
+  "description": "Attribute / class binding type.",
+  "version": "0.1.0",
+  "author": "Trey Shugart <treshugart@gmail.com> (http://treshugart.github.io)",
+  "repository": "git@github.com:skatejs/skatejs-types",
+  "main": "lib/index.js",
+  "ignore": [
+    "**/.*",
```

Comment:
I feel like it might be beneficial to use `"files"` (whitelist) instead of `"ignore"` (blacklist).